crates/bevy_pbr/src/light.rs

```diff
@@ -605,6 +605,17 @@ pub struct NotShadowCaster;
 #[derive(Component, Reflect, Default)]
 #[reflect(Component, Default)]
 pub struct NotShadowReceiver;
+/// Add this component to make a [`Mesh`](bevy_render::mesh::Mesh) not receive shadows
+/// on its diffuse transmission lobe.
+///
+/// Useful, for example, when a thick translucent mesh using [`StandardMaterial::diffuse_transmission`](crate::pbr_material::StandardMaterial::diffuse_transmission)
+/// has a complicated shape that's hard to model via [`StandardMaterial::thickness`](crate::pbr_material::StandardMaterial::thickness), and you'd like
+/// to avoid self-shadows from affecting the transmitted light.
+///
+/// **Note:** Setting [`NotShadowReceiver`] disables both regular and transmitted shadows.
```

Comment:
```
/// **Note:** Setting [`NotShadowReceiver`] disables both regular and transmitted shadows.
```

Lets write this on NotShadowReceiver directly, and link to NotTransmittedShadowReceiver.